lookout/style/format/analyzer.py

```diff
@@ -189,10 +131,10 @@ def check_training_required(
             data_service.get_data(), old_model.ptr, ptr, contents=True, uast=False))
         base_files_by_lang = files_by_language(c.base for c in changes)
         head_files_by_lang = files_by_language(c.head for c in changes)
-        config = cls._load_train_config(config)
+        config = cls._load_config(config)
         for language, head_files in head_files_by_lang.items():
             try:
-                lang_config = config[language]
+                lang_config = config["train"][language]
```

Comment:
Put `config["train"]` to a var

Reply:
So do you want something like
```python
train_config = config["train"]
lang_config = train_config[language]
```
Only `lang_config` is used and there is no more use of `train_config`

Reply:
Yep, lgtm